fenix/app/src/main/java/org/mozilla/fenix/share/AddNewDeviceFragment.kt

```diff
@@ -44,7 +45,7 @@ class AddNewDeviceFragment : Fragment(R.layout.fragment_add_new_device) {
             AlertDialog.Builder(requireContext()).apply {
                 setMessage(R.string.sync_connect_device_dialog)
                 setPositiveButton(R.string.sync_confirmation_button) { dialog, _ -> dialog.cancel() }
-                create()
+                create().withCenterAlignedButtons()
```

Comment:
I would like to confirm if this call it's safe? As it looks like we are calling `withCenterAlignedButtons()` before `show()`